sdk/identity/azure-identity-extensions/src/main/java/com/azure/identity/extensions/implementation/credential/provider/DefaultTokenCredentialProvider.java

```diff
@@ -44,6 +44,7 @@ public TokenCredential get(TokenCredentialProviderOptions options) {
         return resolveTokenCredential(options);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Do you know which API is deprecated?

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/a7d7c6754fe8468789b38000703317b5bbe49e52/sdk/identity/azure-identity-extensions/src/main/java/com/azure/identity/extensions/implementation/credential/provider/DefaultTokenCredentialProvider.java#L71

Reply:
How about we change to the recommended way instead? 
<img width="938" alt="image" src="https://github.com/user-attachments/assets/83a36fa5-95d5-4e11-b6fa-2abe213b2f42" />